go/shared.go

```diff
@@ -0,0 +1,21 @@
+package loyalty
+
+import "github.com/afitz0/customer-loyalty-workflow/status"
+
+type (
+	CustomerInfo struct {
+		CustomerId    string
+		LoyaltyPoints int
+		StatusLevel   int
+		Status        status.Status
+		Name          string
+		Guests        map[string]struct{}
+		AccountActive bool
+	}
+
+	GetStatusResponse struct {
+		Tier          status.Tier
+		Points        int
+		AccountActive bool
+	}
+)
```

Comment:
You don't have to use the `type (` approach btw. I never do. You can just declare top-level, e.g.:

```suggestion
type CustomerInfo struct {
	CustomerId    string
	LoyaltyPoints int
	StatusLevel   int
	Status        status.Status
	Name          string
	Guests        map[string]struct{}
	AccountActive bool
}

type GetStatusResponse struct {
	Tier          status.Tier
	Points        int
	AccountActive bool
}
```